assets/js/components/coverage/CoverageHotspotShowDevicesTab.jsx

```diff
@@ -34,14 +34,25 @@ const columns = [
 ];
 
 export default (props) => {
+  const [page, setPage] = useState(1);
+  const [pageSize, setPageSize] = useState(25);
```

Comment:
this probs shouldn't be in state but instead a const?

Reply:
fixed